### PR TITLE
Stats in ProfileScreen

### PR DESCRIPTION
**Pull request by NBCCF025**
## Motivation
The current statistics screen doesn't show a lot of stats, especialy with the difficulty encountered for displaying the graph. The stats could therefore be regrouped in the profile screen.

## Changes

Figma was synchronized with these changes.
- remove the stats button
- put the stats instead

## Tests
I've tested the following,
- All ProfileScreen() elements are displayed

## Summary
With these changes, we cut an entire screen from the app, further streamlining the UX.

### CODE REVIEW

File: app/src/androidTest/java/com/github/se/signify/ui/screens/profile/ProfileScreenTest.kt [hunk 1]
```diff
@@ -129,4 +125,35 @@ class ProfileScreenTest {
...
+    composeTestRule.onNodeWithTag("QuestsRow").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("QuestsText").performScrollTo().assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag("QuestsText")
+        .performScrollTo()
+        .assertTextEquals("Number of quests achieved :")
+    composeTestRule.onNodeWithTag("Daily").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Daily").performScrollTo().assertTextEquals("Daily")
+    composeTestRule.onNodeWithTag("Weekly").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Weekly").performScrollTo().assertTextEquals("Weekly")
```
Comment (ThomasMPont) [inline app/src/androidTest/java/com/github/se/signify/ui/screens/profile/ProfileScreenTest.kt:157]: Tests should never be checking for exact text.
You've merely moved this test, so it's not your fault. We have hardcoded text assertions everywhere already anyway.
Ideally, these tests would use string resources.

File: app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt [hunk 2]
```diff
@@ -63,11 +64,21 @@ fun ProfileScreen(
...
         }
         val userName = userViewModel.userName.collectAsState()
         val profilePictureUrl = userViewModel.profilePictureUrl.collectAsState()
         val streak = userViewModel.streak.collectAsState()
         val lettersLearned = statsViewModel.lettersLearned.collectAsState()
+        val easy = statsViewModel.easy.collectAsState()
+        val medium = statsViewModel.medium.collectAsState()
+        val hard = statsViewModel.hard.collectAsState()
+        val daily = statsViewModel.daily.collectAsState()
+        val weekly = statsViewModel.weekly.collectAsState()
```
Comment (ThomasMPont) [inline app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt:81]: This really belongs in a data class. Again, it's not for this task.

File: app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt [hunk 3]
```diff
@@ -99,14 +110,31 @@ fun ProfileScreen(
...
+        StatisticsTable(
+            columnTestTag = "QuestsColumn",
+            rowTestTag = "QuestsRow",
+            lineText = questsText,
+            statsTexts = listOf(dailyQuestsText, weeklyQuestsText),
+            statsNumberList = listOf("${daily.value}", "${weekly.value}"),
+            lineTextTestTag = "QuestsText")
       },
   )
 }
```
Comment (ThomasMPont) [inline app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt:140]: ## Requested changes

You really should add some padding down here. Your stats are tangent to the navigation bar.

**Review by ThomasMPont — CHANGES_REQUESTED** [review]
## Overview

This PR is mostly just moves. I don't see it causing any issues. I've tested the app manually as a sanity check. Clean work!

## Requested changes

I've requested a very simple change about missing padding.

## Next steps

The "friends" button is the only remaining square button now. We should move it to the top bar before the end of the project.

File: app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt (hunk 3 above)
Comment (NBCCF025) [reply app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt:140]: I made it in a way that we shouldn't have any screen with their last element glued to the bottom of the screen.

**Review by ThomasMPont — APPROVED** [review]
All good 👍

**Comment by ThomasMPont** [comment]
I've deleted a non-deterministic test on your branch, that was unfairly failing your CI.

**Pull request merged**